nvme.c

```diff
@@ -5227,42 +5300,57 @@ static int sanitize_cmd(int argc, char **argv, struct command *cmd, struct plugi
 	return err;
 }
 
-static int nvme_get_properties(int fd, void **pbar)
+static int nvme_get_single_property(int fd, struct get_reg_config *cfg, __u64 *value)
+{
+	struct nvme_get_property_args args = {
+		.args_size	= sizeof(args),
+		.fd		= fd,
+		.offset		= cfg->offset,
+		.value		= value,
+		.timeout	= NVME_DEFAULT_IOCTL_TIMEOUT,
+	};
+	int err = nvme_get_property(&args);
```

Comment:
new line after the struct defintinon but not a new line after the nvme_get_porperty() call.